resources/v2/01_alter_users_add_columns.sql

```diff
@@ -2,7 +2,9 @@
 
     ALTER TABLE users 
     ADD COLUMN gender ENUM('male','female','others'), 
-    ADD COLUMN updated_at DATETIME DEFAULT CURRENT_TIMESTAMP NOT NULL;
-
+    ADD COLUMN updated_at DATETIME DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    DROP otp, 
+    DROP otp_generated_at, 
+    DROP current_location;
     
```

Comment:
Didn't we merge these changes?

Reply:
Yesterday, in the call we moved that altering operation of users in one file so it's that change.